### DISCUSSION TITLE
Light Panel at GARMIN Edge 830 not shown in Detail

### DISCUSSION THREAD

**Discussion by klausS69** [maca88/SmartBikeLights#62] [Q&A]
My Light panel on Edge 830 does not show the buttons with labels as shown in the instructions.
I have 2 devices connected, a Bontrager ION 200 RT and a GARMIN Varia 515.
Both devices work perfectly so far and with the filter settings as desired.
But the display on the 830's light panel isn't as detailed. I also see both devices in separate columns.
However, the buttons only have letters or numbers as labels.
Is this only possible on the 1030, 1040 etc. with the large displays?

**Comment by telemaxx** [comment]
same to me.
Edge 840 with varia 515

only numbers on the buttons.

rest is working fine

**Comment by maca88** [comment]
You need to create the configuration for your lights by using the Lights Configurator: https://maca88.github.io/SmartBikeLights/
and then copy the configuration value into the "Lights Configuration" app setting by using either Garmin Store or Garmin Express app.
Here are some images for Garmin Express  (select the dots for Smart Bike Lights and not Bike Lights Control as in the image):
![image](https://github.com/maca88/SmartBikeLights/assets/5311349/f3544f84-8912-415f-a9d0-f1e96d7ced21)
![image](https://github.com/maca88/SmartBikeLights/assets/5311349/7253b442-d90c-41bd-bf57-f7b5abb443fc)

For Edge 830 with ION 200 RT and Varia 515 you can use this base configuration (no filters):

#4587520,196641::1##0,73535488::1##7,6:Ion 200 RT:0:16777215!2,:-1,Off:0!1,High:1!1,Medium:2!1,Low:5!1,Day Flash:63!1,Night Flash:62#6,5:Varia 515:0:16777215!2,:-1,Off:0!1,Solid:4!1,Peloton:5!1,Day Flash:7!1,Night Flash:6#0::#0:0#123!:123!#0#0#B3122#2#3#0#0

**Comment by telemaxx** [comment]
Now i it is working.
thanks for you're help.
